app/client/.dockerignore

```diff
@@ -1 +1,3 @@
 node_modules
+.env
```

Comment:
What is this .env used for in app/client? I'm pretty sure this will break our front end. We need to use the root level .env file or setting variables in the docker-compose. If you need help with that let us know!

Reply:
Same probably applies for the src/tests folder entry. Original thoughts were that they weren't wanted in the build, but that might not be the case now.